### DISCUSSION TITLE
rustaceanvim not working autocomplete

### DISCUSSION THREAD

**Discussion by cirqueira-labs** [mrcjkb/rustaceanvim#344] [Q&A]
### Feature description

I made a fresh install of neovim and kickstart and I tried configure the rust-analyzer LSP
I've tried using Mason Install, manul config in init.lua and 'mrcjkb/rustaceanvim'
I have all rust / rustup / cargo bin installed
But, doesnt work completions for Rust
For Lua it's looking great
thanks for help
regards

**Comment by mrcjkb** [comment]
Hey 👋 

What's the output of `:checkhealth rustaceanvim`?

By default, this plugin doesn't support setting up rust-analyzer with mason. You have to configure it manually, see `:h rustaceanvim.mason`.

With rustup (I recommend this over mason), you need to add rust-analyzer (`rustup component add rust-analyzer`)

**Reply by cirqueira-labs** [reply]
Hey @mrcjkb thanks for help ... I tried with and without Mason, adding e running rust-analyzer by rustup run.
But no autocomplete are shown.

**Reply by mrcjkb** [reply]
What's the output of `:checkhealth rustaceanvim`?
